### PR TITLE
Probleme : rack asset creation broken

### PR DESCRIPTION
**Pull request by nicodav**
Solution : 
For a new rack, the sse make a call to fty-metric-cache, but if fty-metric-cache does not known this new asset, it return result = null.
So add a test if result from metric-cache is null or not.

### CODE REVIEW

File: src/shared/utils_json.cc [hunk 1]
```diff
@@ -85,11 +85,17 @@ s_rack_realpower_nominal(
   zstr_free(&uuid);
 
   char *result = zmsg_popstr(msg);
-  if (!streq(result, "OK"))
+  if (NULL == result || !streq(result, "OK"))
   {
     log_warning("Error reply for device '%s', result=%s", name.c_str(), result);
-    zstr_free(&result);
-    zmsg_destroy(&msg);
+    if (NULL != result)
```
Comment (EldoreiJK) [inline src/shared/utils_json.cc]: Please use consistent code style, either check for (var != NULL) or just use (var), not both.

File: src/shared/utils_json.cc [hunk 2]
```diff
@@ -85,11 +85,17 @@ s_rack_realpower_nominal(
...
+    {
+      zstr_free(&result);
+    }
+    if (msg)
```
Comment (EldoreiJK) [inline src/shared/utils_json.cc]: Please use consistent code style, either check for (var != NULL) or just use (var), not both.

Comment (jimklimov) [reply src/shared/utils_json.cc]: Indeed, same style especially in short sequence, is good.

Pedantically speaking, pointers should be compared to NULL explicitly (at least per older standards as they did not require that NULL is numeric zero and some architectures ages ago did have other special values meaning NULL - so `if (ptrname)` did not do what we expect "intuitively" nowadays). Additionally, comparing to NULL adds a bit of type-sanity checking at compile time (avoid using numeric variables where pointers are expected).
